frontend/awx/views/jobs/hooks/useJobHeaderActions.tsx

```diff
@@ -0,0 +1,51 @@
+import { ButtonVariant } from '@patternfly/react-core';
+import { MinusCircleIcon, RocketIcon } from '@patternfly/react-icons';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { IPageAction, PageActionSelection, PageActionType } from '../../../../../framework';
+import { UnifiedJob } from '../../../interfaces/UnifiedJob';
+import { isJobRunning } from '../jobUtils';
+import { useCancelJobs } from './useCancelJobs';
+import { useRelaunchJob } from './useRelaunchJob';
+
+export function useJobHeaderActions(onComplete: (jobs: UnifiedJob[]) => void) {
+  const { t } = useTranslation();
+  const cancelJobs = useCancelJobs(onComplete);
+  const relaunchJob = useRelaunchJob();
+
+  return useMemo<IPageAction<UnifiedJob>[]>(() => {
+    const cannotCancelJob = (job: UnifiedJob) => {
+      if (!isJobRunning(job.status))
+        return t(`The job cannot be canceled because it is not running`);
+      else if (!job.summary_fields.user_capabilities.start)
+        return t(`The job cannot be canceled due to insufficient permission`);
+      else return '';
+    };
+
+    const actions: IPageAction<UnifiedJob>[] = [
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Single,
+        variant: ButtonVariant.secondary,
+        isPinned: true,
+        icon: RocketIcon,
+        label: t(`Relaunch job`),
```

Comment:
Should we let the user relaunch a job if it's still running/pending? It can lead to creating a lot of the same pending jobs.

![Screenshot from 2024-03-12 14-49-41](https://github.com/ansible/ansible-ui/assets/19647757/694af676-18e4-41b7-8cfa-98c2e1d10d60)

Reply:
@jerabekjiri It looks like the current (through row actions) and the old UI have the ability to relaunch when pending/running. 

I'd imagine the behavior would want to be transferred over to the new UI.